exporter/f5cloudexporter/go.mod

```diff
@@ -1,6 +1,6 @@
-module github.com/gramidt/opentelemetry-collector-contrib/exporter/f5cloudexporter
+module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/f5cloudexporter
 
-go 1.15
+go 1.14
```

Comment:
We use go 1.15 everywhere I believe

Reply:
The main go.mod for both the core collector and contrib collector have 1.14 defined, as well as most of the components go.mods in the contrib have 1.14 defined.

Reply:
I'm always happy to help update all of these if that is what we would like to do.

Reply:
Touche :)) let's do that later then